experiments/openebs/openebs-target-network-delay/cstor_target_network_delay.yaml

```diff
@@ -1,18 +1,19 @@
---- 
-- name: Pick a cStor target pod belonging to the PV
-  shell: >
-    kubectl get pods -l {{ cstor_target_pod_label }}
-    -n {{ openebs_ns }}  -o jsonpath='{.items[?(@.metadata.labels.openebs\.io/persistent-volume=="{{ pv_name }}")].metadata.name}'
-  args:
-    executable: /bin/bash
-  register: cstor_target_pod
+- block:
+  - name: Pick a cStor target pod belonging to the PV
+    shell: >
+      set -o pipefail &&
```

Comment:
Why we are using `set -o pipefail`  here?

Reply:
It's a necessary thing to be added for passing the ansible lint
- it is used for failed the task if any command in the shell exits with other than `0`